fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/SketchSetOperationMaker.java

```diff
@@ -27,7 +27,7 @@
 
     private static final int DEPENDENT_METRICS_REQUIRED = 2;
 
-    final SketchSetOperationPostAggFunction function;
+    private final SketchSetOperationPostAggFunction function;
```

Comment:
Probably should call this change out in the change log. It's unlikely this was extended, but just in case...